core/src/main/java/io/questdb/MessageBusImpl.java

```diff
@@ -200,6 +209,18 @@ public void close() {
         Misc.free(pageFrameReduceQueue);
     }
 
+    @Override
+    public void reset() {
```

Comment:
nit: this test should be marked as `@TestOnly`. I'd also keep it in the implementation class and avoid exposing it in the interface. We call it in `CairoEngine` which is guaranteed to work with a `MessageBusImpl`, so having it as a public method in `MessageBusImpl` should be enough.

Reply:
agree, let me change it